pkgs/top-level/python-packages.nix

```diff
@@ -27121,13 +27121,15 @@ in modules // {
     };
   };
 
-  udiskie = buildPythonPackage rec {
-    version = "1.4.8";
+  udiskie = buildPythonApplication rec {
```

Comment:
Added a commit that moves the package to `applications/misc/udiskie`. I kept a reference in `pythonPackages` for backwards compatibility. I'm not sure whether that is necessary, though…